### ISSUE TITLE
Keep call running when refresh page

### ISSUE THREAD

**Issue by himanshutecstub** [ringcentral/ringcentral-embeddable#587]
Hello,
I use https://github.com/ringcentral/ringcentral-embeddable widget code in my HTML web page which is working fine with call send & receive.
When any call running and i press browser refresh than call ending.
Is there any way to keep widget open or can't refresh when call running.

Thanks

**Comment by embbnux** [comment]
hi @himanshutecstub, you can try [the popup feature](https://github.com/ringcentral/ringcentral-embeddable/blob/master/docs/popup-window.md). As the widget is a web page, everything will be destroyed by Browser when user close or refresh page. We can only keep the call when user have call at popup window, and keep popup window opened.

**Comment by himanshutecstub** [comment]
Yes i enable this feature in HTML page and popup icon will open new popup window but when any call running than can't open popup.

**Comment by embbnux** [comment]
yeah, it is by design now. Currently app will end the call if user open the popup window. We are still finding some way to improve it
